### PR TITLE
Rename StopEbola into Emergency Information

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony @bruskiza Ready for review

File: springboard_iogt/templates/base.jinja2.orig [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      dir="{{ language|language_direction }}">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link rel="shortcut icon" type="image/x-icon" href="{{'springboard:static/img/favicon.ico'|static_url}}" />
+    <title>{{ title|default(_("The Internet of Good Things")) }}</title>
+    <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;" />
+    <meta name="description" content="{{ description|default("") }}" />
+    <meta name="keywords" content="{{ keywords|default("") }}" />
+    <link href="{{'springboard:static/css/style.css'|static_url}}" rel="stylesheet" type="text/css" />
+<<<<<<< HEAD
```
Comment (bruskiza) [inline springboard_iogt/templates/base.jinja2.orig]: This is conflicted. Please can this be resolved.

File: dump.rdb [hunk 2]
```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```
Comment (bruskiza) [inline dump.rdb]: Please add to .gitignore.

**Comment by g1039** [comment]
@bruskiza @miltontony Ready for re-review

File: dump.rdb (hunk 2 above)
Comment (bruskiza) [inline dump.rdb]: Please add `dump.rdb` to `.gitignore` and remove `dump.rdb` from this commit.

**Comment by g1039** [comment]
@bruskiza ready for review

File: springboard_iogt/templates/base.jinja2.orig [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        {% endblock %}
+    </div>
+
+    <div id="footer">
+        <div>&copy; {% trans %}The Internet of Good Things{% endtrans %}</div>
+        <div><span><a href="{{ 'about'|route_url }}">{{ _('About Internet of Good Things') }}</a></span></div>
+    </div>
+
+</body>
+</html>
```
Comment (bruskiza) [inline springboard_iogt/templates/base.jinja2.orig]: Please remove this file.
It's not required.

**Comment by bruskiza** [comment]
looks good to me.
@miltontony can u review please?

**Comment by miltontony** [comment]
:+1:

**Pull request merged**